integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -837,7 +868,11 @@ fn test_resharding_v3_drop_chunks_all() {
 fn test_resharding_v3_resharding_block_in_fork() {
     test_resharding_v3_base(
         TestReshardingParametersBuilder::default()
-            .num_validators(1)
+            .num_clients(1)
+            .num_producers(1)
+            .num_validators(0)
+            .num_rpcs(0)
+            .num_archivals(0)
```

Comment:
What is the motivation to disable rpc and archival?

Reply:
It's simpler to trigger forks with specific combinations of blocks height with only one node